snowpack/src/index.ts

```diff
@@ -105,15 +105,15 @@ export async function cli(args: string[]) {
 
   if (cmd === 'build') {
     await buildCommand(commandOptions);
-    return;
+    return process.exit(0);
```

Comment:
This is the one that matters (and has likelihood of spawning subprocesses) -- the others were added for consistency.